c10/core/CPUAllocator.cpp

```diff
@@ -162,6 +246,8 @@ at::Allocator* GetDefaultCPUAllocator() {
 
 REGISTER_ALLOCATOR(DeviceType::CPU, &g_cpu_alloc);
 
+#endif /* C10_Mobile */
+
```

Comment:
This section (the whole ifdef-else-endif block, I can't highlight the whole thing in github) is very easy to understand and should be easy to modify safely.  The part above the ifdef could possibly be moved to another .cpp file since `GetDefaultMobileCPUAllocator` is already declared in a header, but given how short it is, I don't think it's necessary.